web-app/src/components/Leave/index.jsx

```diff
@@ -2,16 +2,36 @@ import React from 'react';
 import { PropTypes as PT } from 'prop-types';
 import container from './container';
 import { Card } from '../common/';
+import { Button } from '../common'
+import { Container, Content } from '../common/Modal/styled';
```

Comment:
It'd be better if we exported the Modal as its own component from `common`, rather than grabbing the styled components that it would use - if that makes sense.